chia/cmds/beta.py

```diff
@@ -29,8 +32,9 @@ def beta_cmd() -> None:
 
 @beta_cmd.command("configure", help="Configure the beta test mode parameters")
 @click.option("-p", "--path", help="The beta mode root path", type=str, required=False)
+@click.option("-i", "--interval", help="System metrics will be logged based on this interval", type=int, required=False)
 @click.pass_context
```

Comment:
what do you  think about making the interval be the default 
if not specified and losing the prompt ?